### DISCUSSION TITLE
[BUG] Cada vez que grabo una sm me arroja un error, pero no sale nada en la consola

### DISCUSSION THREAD

**Discussion by NicolasPLG** [Cotalker/documentation#164] [4. Bug Reports] [answered]
Cuando edito o grabo en una SM me arroja un error, veo en la consola y no hay nada, quizás sea un mensaje pegado.

Dejo un SS
Gracias
![image](https://user-images.githubusercontent.com/61419471/117719901-fa317000-b1ab-11eb-96f5-b0570297c8b8.png)

**Comment by rafaelwalter** [comment, marked as answer]
@NicolasPLG cuales cambios intentabas hacer en la SM?

**Reply by rafaelwalter** [reply]
@NicolasPLG identificamos un error en nuestra plataforma de crash reports, creamos un ticket interno para solucionarlo. Gracias!

**Reply by NicolasPLG** [reply]
gracias @rafaelwalter cerraré el ticket
